WordPress/src/main/java/org/wordpress/android/models/AccountModel.java

```diff
@@ -20,6 +23,7 @@
     private String mFirstName;
     private String mLastName;
     private String mAboutMe;
+    private Date mDate;
```

Comment:
Can we use a better name? `mDateCreated` - I know the REST API replies with a `date` key :sob: